packages/mco/components/mco-dashboard/data-policy/alert-card/alert-item.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import {
+  getAlertDescription,
+  getAlertMessage,
+  getAlertName,
+  getAlertSummary,
+  getAlertTime,
+  getSeverityIcon,
+} from '@odf/shared/alert/utils';
+import { Timestamp } from '@odf/shared/details-page/timestamp';
+import { getAlertSeverity } from '@odf/shared/utils';
+import { Alert } from '@openshift-console/dynamic-plugin-sdk';
+import './alert-item.scss';
+
+export const StatusItem: React.FC<StatusItemProps> = ({
+  name,
+  Icon,
+  timestamp,
+  message,
+}) => {
+  return (
+    <div className="mco-status-card__alert-item">
+      <div className="mco-status-card__alert-item-icon mco-dashboard-icon">
+        <Icon />
+      </div>
+      <div className="mco-status-card__alert-item-text">
+        <div className="mco-status-card__alert-item-message">
+          {name && (
+            <span className="mco-status-card__alert-item-header">{name}</span>
+          )}
+          <div
+            className="mco-health-card__alert-item-timestamp mco-status-card__health-item-text text-secondary"
+            data-test="timestamp"
+          >
+            {timestamp && <Timestamp simple timestamp={timestamp} />}
+          </div>
+          <span className="mco-status-card__health-item-text">{message}</span>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+const AlertItem: React.FC<AlertItemProps> = ({ alert }) => {
```

Comment:
```suggestion
const AlertItem: React.FC<React.ComponentProps<typeof AlertItemSDK>> = ({ alert }) => {
```
import the `AlertItem` from SDK as `AlertItemSDK`